mock-user-management.js

```diff
@@ -77,7 +77,7 @@ function storeInWallet(verifiablePresentation) {
 
   // base64 encode the serialized contents (verifiable presentations)
   const serialized = btoa(JSON.stringify(walletContents));
-  Cookies.set('walletContents', serialized, {path: '', secure: true, sameSite: 'None'});
+  localStorage.setItem('walletContents', serialized);
```

Comment:
I'm pretty sure we could just use JSON now as well. That would make the data a bit more readable / easier to debug when using devtools to read local storage directly. IOW, we shouldn't need to base64-encode the data to store in `localStorage`, that was only needed to store as a cookie.

Reply:
Added a follow up commit to handle this